tools/build_versions.py

```diff
@@ -1,22 +1,26 @@
 #!/usr/bin/env python
 
 from pathlib import Path
-import requirements
 import pkg_resources
+import re
 
 
 def main():
     requirements_dir = Path(__file__).parent / '..' / 'requirements'
 
     for requirement_file in requirements_dir.glob('*.txt'):
         print(requirement_file.name)
-        with open(str(requirement_file), 'r') as fd:
-            for req in requirements.parse(fd):
+        with open(str(requirement_file), 'r') as f:
+            for req in f:
+                req = req.strip()
+                if req[0] == '#':
```

Comment:
This will give an IndexError if we encounter a blank line. On the one hand, I suggest replacing with `if req.startswith('#')`. On the other, I'll take this opportunity to point out that robustness to this kind of error is the kind of thing that a dependency buys you.